src/jobs/push-to-app-collection.yaml

```diff
@@ -4,19 +4,24 @@ description: >
 executor: architect
 parameters:
   app_name:
-    description: "Name of the application"
+    description: "Name of the application."
     type: "string"
   app_catalog:
-    description: "Name of the catalog CR where the application belongs"
+    description: "Name of the catalog CR where the application belongs."
     type: "string"
     default: "control-plane-catalog"
   app_collection_repo:
-    description: "Name of the app collection repository"
+    description: "Name of the app collection repository."
     type: "string"
+  unique:
+    description: "Whether the previous App versions in the collection should be replaced with this one. Any non-empty string means true."
```

Comment:
It'd be useful to document _when_ to use this. e.g: this is useful for applications which we don't support running multiple versions of (e.g: gatekeeper)